### PR TITLE
Added issue templates

### PR DESCRIPTION
**Pull request by Nicklason**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+For errors, add a screenshot of the error. If that is not possible, then see the error log.
+
+Log files are located at `/tf2-automatic/logs`. There are three log files:
+* <steam account name>-<date>.log - Default log file
+* <steam account name>.error.log - Error log
+* <steam account name>.trade.log - Trade log
+-->
+
+**Additional context**
```
Comment (Gobot1234) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think this should just be additional notes

Comment (Nicklason) [reply .github/ISSUE_TEMPLATE/bug_report.md]: No. The logs should always be included in a bug report.

Comment (Nicklason) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Or you mean that the title should be changed to "Additional notes"?

Comment (ZeusJunior) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Probably

Comment (Nicklason) [reply .github/ISSUE_TEMPLATE/bug_report.md]: If that's the case then no, it is fine

**Pull request merged**